### PR TITLE
MGMT-3976: Embed ramdisk placeholder in template

### PR DESCRIPTION
**Pull request by danielerez**
* Added 'assisted_installer_custom.img' file to images folder.
* Modified config files in the template to include it in initrd.

### CODE REVIEW

File: internal/isoeditor/rhcos.go [hunk 1]
```diff
@@ -96,6 +104,33 @@ func (e *rhcosEditor) CreateClusterMinimalISO(ignition string, staticIPConfig st
...
+
+func (e *rhcosEditor) createImagePlaceholder(imagePath string) error {
+	f, err := os.Create(e.isoHandler.ExtractedPath(imagePath))
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	data := make([]byte, initrdPaddingLength)
+	_, err = f.Write(data)
```
Comment (carbonin) [inline internal/isoeditor/rhcos.go:126]: I think `f.Truncate(initrdPaddingLength)` would be better than maintaining a 256k array in memory.

Comment (danielerez) [reply internal/isoeditor/rhcos.go:126]: right, should be good enough.

**Review by carbonin — APPROVED** [review]

**Pull request merged**